### PR TITLE
Docs: alertmanager datasource

### PR DESCRIPTION
**Pull request by domasx2**
**What this PR does / why we need it**:
A docs entry for the alertmanager datasource that is going to be introduced with 8.
As discussed in slack, trargeting feature branch for docs for now

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/datasources/alertmanager.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
+Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+>**Note:** New in Grafana 8.0.
+
+>**Note:** Currently only [Cortex Alertmanager](https://cortexmetrics.io/docs/proposals/scalable-alertmanager/) is supported.
+
+>**Note:** It is presently in alpha and not accessible unless alpha plugins are [enabled in Grafana settings](https://grafana.com/docs/grafana/latest/administration/configuration/#enable_alpha).
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:18]:
```suggestion
It is presently in alpha and not accessible unless [alpha plugins are enabled in Grafana settings](https://grafana.com/docs/grafana/latest/administration/configuration/#enable_alpha).
```

File: docs/sources/datasources/alertmanager.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Provision the Alertmanager data source
+
+You can configure data sources using config files with Grafana's provisioning system. Read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}).
+
+Here is an example for provisioning this datasource:
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:24]:
```suggestion
Here is an example for provisioning the Alertmanager data source:
```

File: docs/sources/datasources/alertmanager.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
++++
+
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
+Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+>**Note:** New in Grafana 8.0.
+
+>**Note:** Currently only [Cortex Alertmanager](https://cortexmetrics.io/docs/proposals/scalable-alertmanager/) is supported.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:16]: Is this Cortex Alertmanager or Prometheus?

File: docs/sources/datasources/alertmanager.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+>**Note:** It is presently in alpha and not accessible unless alpha plugins are [enabled in Grafana settings](https://grafana.com/docs/grafana/latest/administration/configuration/#enable_alpha).
+
+## Provision the Alertmanager data source
+
+You can configure data sources using config files with Grafana's provisioning system. Read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:22]:
```suggestion
You can configure the Alertmanager data sources using config files with Grafana's provisioning system. For more information on how it works and the settings available, refer to the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}).
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added change requests.

File: docs/sources/datasources/alertmanager.md (hunk 3 above)
Comment (domasx2) [reply docs/sources/datasources/alertmanager.md:16]: both in a way, as in Cortex implementation of Prometheus Alertmanager :sweat_smile:

Comment (achatterjee-grafana) [reply docs/sources/datasources/alertmanager.md:16]: Can we make this little clear, then we should be good to go. Thanks!

Comment (domasx2) [reply docs/sources/datasources/alertmanager.md]: Updated to "Cortex implementation of Prometheus Alertmanager"

File: docs/sources/datasources/alertmanager.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+title = "Alertmanager"
+description = "Guide for using Alertmanager in Grafana"
+keywords = ["grafana", "prometheus", "guide"]
+aliases = ["/docs/grafana/latest/features/datasources/alertmanager"]
+weight = 1300
++++
+
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:11]:
```suggestion
Grafana includes built-in support for Prometheus Alertmanager. Once you add it as a data source, you can use the [Grafana unified alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points as well as notification policies. A drop down option in these pages allows you to switch between Grafana and any configured Alertmanager data sources 

The Alertmanager data source cannot return graph data and is not available in panels.
```

File: docs/sources/datasources/alertmanager.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+>**Note:** New in Grafana 8.0.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:14]: Remove note, as these docs are for 8.0.

File: docs/sources/datasources/alertmanager.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+>**Note:** Currently only [Cortex implementation of Prometheus alertmanager](https://cortexmetrics.io/docs/proposals/scalable-alertmanager/) is supported.
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:16]:
```suggestion
>**Note:** Currently only the [Cortex implementation of Prometheus alertmanager](https://cortexmetrics.io/docs/proposals/scalable-alertmanager/) is supported. 
```

File: docs/sources/datasources/alertmanager.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+# Alertmanager data source
+
+Grafana includes built-in support for Prometheus Alertmanager. Add it as a datasource and you will be able to use the [Grafana alerting UI](https://grafana.com/docs/grafana/latest/alerting/) to manage silences, contact points and notification policies. A dropdown will appear in these pages to allow you to switch between Grafana and any configured alertmanager data sources This data source cannot return graph data and is not available in panels.
+Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
+
+>**Note:** New in Grafana 8.0.
+
+>**Note:** Currently only [Cortex implementation of Prometheus alertmanager](https://cortexmetrics.io/docs/proposals/scalable-alertmanager/) is supported. 
+
+It is presently in alpha and not accessible unless [alpha plugins are enabled in Grafana settings](https://grafana.com/docs/grafana/latest/administration/configuration/#enable_alpha).
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:18]: This should be the second line in the first paragraph.

File: docs/sources/datasources/alertmanager.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+## Provision the Alertmanager data source
+
+You can configure the Alertmanager data sources using config files with Grafana's provisioning system. For more information on how it works and the settings available, refer to the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/alertmanager.md:22]:
```suggestion
Configure the Alertmanager data sources by updating Grafana's configuration files. For more information on how it works and the settings available, refer to the [provisioning docs page]({{< relref "../administration/provisioning/#datasources" >}}).
```

**Review by achatterjee-grafana — CHANGES_REQUESTED** [review]
Added copy edit changes.

File: docs/sources/datasources/alertmanager.md (hunk 5 above)
Comment (domasx2) [reply docs/sources/datasources/alertmanager.md:11]: Are we publically calling this "unified" alerting? I thought "unified" is an internal designation

File: docs/sources/datasources/alertmanager.md (hunk 6 above)
Comment (oddlittlebird) [reply docs/sources/datasources/alertmanager.md:14]: How big a deal is this? Should we mention it in What's new in Grafana 8.0? @pkolyvas, thoughts?

Comment (oddlittlebird) [reply docs/sources/datasources/alertmanager.md:14]: Generally if we are adding a whole new topic to document something, I want it called out in the What's new. However, we have previously not done this (added docs) for alpha functionality.

Comment (achatterjee-grafana) [reply docs/sources/datasources/alertmanager.md:14]: This Alertmanager helps with the new alerting. So, a mention within that topic in What's New is probably the best.

Comment (domasx2) [reply docs/sources/datasources/alertmanager.md:14]: This is part of the new alerting system, not really a stand-alone thing. I don't see an entry for new alerting at all in what's new for 8.0, and I don't think it makes sense to tackle it in the scope of this PR. Is that OK?

File: docs/sources/datasources/alertmanager.md (hunk 5 above)
Comment (domasx2) [reply docs/sources/datasources/alertmanager.md]: updated, but as per slack discussion, w/o "unified" word

File: docs/sources/datasources/alertmanager.md (hunk 8 above)
Comment (domasx2) [reply docs/sources/datasources/alertmanager.md]: done!

File: docs/sources/datasources/alertmanager.md (hunk 6 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/alertmanager.md:13]: > This is part of the new alerting system, not really a stand-alone thing. I don't see an entry for new alerting at all in what's new for 8.0, and I don't think it makes sense to tackle it in the scope of this PR. Is that OK?

Yes

Comment (oddlittlebird) [reply docs/sources/datasources/alertmanager.md:13]: Ah, gotcha. @achatterjee-grafana, please make sure it is linked in the What's new doc when the alerting feature entry is updated.

Comment (achatterjee-grafana) [reply docs/sources/datasources/alertmanager.md:13]: Will add this as a new data source.

**Review by achatterjee-grafana — APPROVED** [review]

**Pull request merged**